.eslintrc.js

```diff
@@ -18,6 +19,7 @@ module.exports = {
     'eslint:recommended',
     'plugin:eslint-comments/recommended',
     'plugin:eslint-plugin/all',
+    'plugin:jest/all',
```

Comment:
are we sure we want **all** rules instead of `recommended`?

Reply:
Switched to enabling the `recommended` and `style` configurations plus ~15 optional rules.